package.json

```diff
@@ -25,8 +25,8 @@
   "author": "Halil Bilir <hbilir@gmail.com>",
   "homepage": "https://github.com/halilb/react-native-photo-browser#readme",
   "dependencies": {
+    "prop-types": "15.5.10",
     "react-native-iphone-x-helper": "^1.2.1",
-    "react-native-progress": "^3.2.0",
-    "prop-types": "15.5.10"
+    "react-native-progress": "^3.2.0"
```

Comment:
It seems nothing has actullay changed in `package.json`. Could you revert those changes?

Reply:
Reverted